docs/rules/no-direct-import.md

```diff
@@ -0,0 +1,22 @@
+# Do not import modules directly
+
+Do not import the core Node.js modules directly.
```

Comment:
This is not correct. It doesn't apply to all core Node.js modules, just some, and not even just core modules.